python/cucim/src/localtest.py

```diff
@@ -109,34 +110,34 @@ def load_tile_cucim(slide, start_loc, tile_size):
 
 openslide_tot_time = 0
 cucim_tot_time = 0
-for num_workers in range(1, num_threads + 1):
+for num_workers in (num_threads,):  # range(1, num_threads + 1):
```

Comment:
Why is this commented?

Reply:
It is modified during the local test (since I didn't want to execute it for all available thread # configurations.)
Reverted it.